realgud/common/track.el

```diff
@@ -193,6 +210,9 @@ evaluating (realgud-cmdbuf-info-loc-regexp realgud-cmdbuf-info)"
 	 (cmdbuf (or opt-cmdbuf (current-buffer)))
 	 )
     (unless (realgud:track-complain-if-not-in-cmd-buffer cmdbuf t)
+      (if (realgud:eval-command-p text)
```

Comment:
I could put this behind a custom variable that a user would have to turn on so it wouldn't be the default behavior, up to you.

Reply:
Let's leave it simple for now. Can do that later there is prevailing opinion to do so.